quarantine.yaml

```diff
@@ -0,0 +1,20 @@
+# list of quarantined tests
```

Comment:
Just a nitpick - probably we can move `quarantine.yaml` to `tests/` directory so we won't overwhelm zephyr home directory?

Reply:
This file will be removed from this dir and will probably end in docs as an example. For now, it just serves as a showcase to play with for the review purpose.